### DISCUSSION TITLE
unable to sign in to copilot

### DISCUSSION THREAD

**Discussion by Dev-Paccore** [community/community#18431] [Projects and Issues]
GitHub Copilot is already enabled on your account! [Install GitHub Copilot](https://github.com/github/copilot-docs) now.
still im unable to access the features of it.

**Comment by Dev-Paccore** [comment]
console message:
[INFO] [auth] [2022-06-12T12:40:51.048Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-06-12T12:40:51.050Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
[ERROR] [auth] [2022-06-12T12:40:55.404Z] Failed to send notification result to GitHub: 403

**Comment by Dev-Paccore** [comment]
Signed in in visual studio and copilot aswell, there is no luck

**Reply by mtmfahathdev** [reply]
@Dev-Paccore Try to uninstall the plugin and install it again, Also make sure the copilot activated account and your local config are same 👍🏽
